Makefile

```diff
@@ -115,6 +115,10 @@ coverage: python
 functests: build/manifest.json python
 	@tox -qe functests
 
+.PHONY: functests-backwards
+functests-backwards: build/manifest.json python
+	@tox -qe functests -- tests/functional --reverse
+
```

Comment:
I think we could probably do without this, especially since it'll need to be copied to every project and kept up to date in every project.

I don't imagine that `make functests-backwards` is a command that most developers will be using often, or ever. The intention is to have them run backwards on CI and *not* in dev. The `Makefile` convenience commands are just meant to be a minimal set of commands that the whole team (including the frontend team) use frequently. For more specific commands we just drop down to running `tox` directly (or the frontend team just runs `gulp` directly etc).

The only time I can imagine wanting to run them backwards locally is to reproduce a failure on CI. In that case if you can't remember how to run the functests in reverse you can look in the GitHub Actions workflow file, but you should just be able to type `reverse` and then use the history search feature in your shell.